src/OrganisationRegistry.ElasticSearch/Organisations/OrganisationDocument.cs

```diff
@@ -1,17 +1,38 @@
 namespace OrganisationRegistry.ElasticSearch.Organisations
 {
-    using Nest;
     using System;
     using System.Collections.Generic;
     using Common;
+    using Nest;
 
     public partial class OrganisationDocument : IDocument
     {
-        public int ChangeId { get; set; }
-        public DateTimeOffset ChangeTime { get; set; }
-        public Guid Id { get; set; }
+        public OrganisationDocument()
+        {
+            Name = string.Empty;
+            OvoNumber = string.Empty;
+            ShortName = string.Empty;
+            Description = string.Empty;
+            KboNumber = string.Empty;
+            Purposes = new List<Purpose>();
+            Labels = new List<OrganisationLabel>();
+            Keys = new List<OrganisationKey>();
+            Contacts = new List<OrganisationContact>();
+            OrganisationClassifications = new List<OrganisationOrganisationClassification>();
+            Functions = new List<OrganisationFunction>();
+            Relations = new List<OrganisationRelation>();
+            Capacities = new List<OrganisationCapacity>();
+            Parents = new List<OrganisationParent>();
+            FormalFrameworks = new List<OrganisationFormalFramework>();
+            Buildings = new List<OrganisationBuilding>();
+            Locations = new List<OrganisationLocation>();
+            Bodies = new List<OrganisationBody>();
+            BankAccounts = new List<OrganisationBankAccount>();
+            OpeningHours = new List<OrganisationOpeningHour>();
+            Regulations = new List<OrganisationRegulation>();
+            Validity = new Period(); // default: infinitely valid
```

Comment:
maybe change to 'Period.Infinity' ? -> then the comment can go

Reply:
I think the comment can go even without a code change, but I don't mind making it more explicit :+1: